src/NServiceBus.Metrics/Probes/DurationProbe.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using NServiceBus;
+
+class DurationProbe : Probe, IDurationProbe
+{
+    public DurationProbe(string name, string description) : base(name, description)
+    {
+    }
+
+    public void Register(Action<TimeSpan> observer)
+    {
+        observers += observer;
```

Comment:
Not sure it is worth discussing but isn't this keeping a strong reference to the observer? Could this leak?